source/2019-09-20-ember-3-13-released.md

```diff
@@ -133,12 +151,35 @@ While it is recommended to keep Ember CLI versions in sync with Ember and Ember
 
 No new features in Ember CLI 3.13.
 
-
 ---
 
 For more details on the changes in Ember CLI 3.13 and detailed upgrade
 instructions, please review the [Ember CLI  3.13.0 release page](https://github.com/ember-cli/ember-cli/releases/tag/v3.13.0).
 
+## Ember 3.12 is an LTS Release
+
+If you are upgrading from the previous LTS version (3.8) to 3.12 then you are getting access to these features:
+
+* `router` service 
+* Support for native classes
+* The `array` helper
+* Element Modifier Manager (and modifiers in general)
+* Nested angle bracket component invocation
+* Angle bracket invocation for built-in components
+* Support for native decorators
+* The `{{fn}}` and `{{on}}` modifiers
+
+## 3.14 Beta
+
+This release kicks off the 3.14 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
```

Comment:
```suggestion

Among other things, the [`@model` named argument](https://github.com/emberjs/rfcs/blob/master/text/0523-model-argument-for-route-templates.md) will be available to route templates. While not officially an "Octane feature", this feature will available for the first time in the 3.14 release, which coincides with the Octane release. Since the Octane guides and documentation are targeting the same version, you will see them using this feature, but please not that the feature is not available in the 3.13 Octane preview.

This beta release also included a new [debug render tree API](https://github.com/emberjs/ember.js/pull/18372), which will be integrated into the Ember Inspector sometime during the beta cycle to give it full Octane support.
```